### DISCUSSION TITLE
How to tell Keka to extract into an existing folder ?

### DISCUSSION THREAD

**Discussion by palijn** [aonez/Keka#1253] [Q&A]
Hi,
I have Google Takeout archives to extract (ZIP). The Google Photos archives are strangely organized, in that they all contain a Takeout/Google Photos/ folder and each archive keeps populating the one single Google Photos folder that is expected at the end of the process.
However, any time I point Keka to the folder in which the existing (first one) Takeout folder is, Keka dutifully creates a "Takeout 1", "Takeout 2" etc folder, which I have to then manually merge with the first one.
Is there a way to tell Keka to extract its paths into the existing paths such as everything immediately ends up in the right place ?
Thanks for your help
Thierry

**Comment by morby** [comment]
Perhaps this problem is specific to Google Drive but I'm experiencing much the same. For example, when one downloads three gigabytes of photos using Drive's 'Download as ZIP' option, Google Drive will dutifully break the folder up into multiple ZIPs, none of which contain all the files but they're all named the same inside the ZIP. As the primary poster mentioned, one has to: 

- Extract all zip archives into folders,
- Open and select all the files and folders from folder number two,
- Option-drag the the selected contents into folder number one,
- Click 'Apply to All' then 'Merge' in the dialog that pops up,
- After the copy process concludes, delete folder number two as it is now redundant,
- Repeat from step two for every subsequent folder.

Not so bad when its two or three gigs because one only receives a few ZIP archives but when you get into the double gigabyte digits, it gets pretty crazy. If Keka had something like WinRAR's ability to extract into one folder or just merge the contents as it extracts them, it would ease this kind of scenario.

If Keka already has this ability, I do apologize but I haven't been able to sleuth out the feature or stumble onto the documentation.
